docs/guides/make_bootable_media.md

```diff
@@ -0,0 +1,24 @@
+# Make Bootable Media
+How to image a drive using your Mark II's "Make Bootable Media" skill.
+
+## Requires
+Two USB drives, or a USB drive and an SSD drive, or two SSD drives with supplemental power to one of them. Any data on the secondary drive will be erased by this process. 
+
+## Steps
+1. Start with your machine booted up, with the boot drive in the upper left USB port.
+2. Plug in your secondary drive in the upper right USB port. 
+3. Say "Hey Neon, make bootable media".
+4. Follow the prompts.
+5. When finished, shut down your Mark II and unplug the power. 
+6. Unplug the boot drive you were using from the left USB port and set it aside. 
+7. Move the second boot drive from the right hand port to the left hand port. 
+8. Plug power back in to your Mark II. 
+9. Congratulations, you're on the new version! 
+10. If you would like to use the new version on your original boot drive, simply repeat this process.
```

Comment:
consider removing or moving these to a different section; if we parse these into spoken help I think it would be confusing to state the process is complete as part of the instructions